app/views/user_mailer/missing.erb

```diff
@@ -0,0 +1,3 @@
+Your account doesn't exist. Please , contact with your administrator
```

Comment:
`Please ,` -> `Please,` maybe? Also '.' at the end.

Reply:
error messages are using I18n, should we use it also in this templates?

Reply:
Also, 'contact with your administrator' -> `contact your administrator`

Reply:
is my condominium. haha!! sorry